test/unit/lib/executor/driver-handler-tests.js

```diff
@@ -99,6 +99,82 @@ describe('lib/executor/driver-handler.js', function() {
         });
     });
 
+    describe('_navigateToAndModifyObject', function() {
+        let webdriver;
```

Comment:
Please prune the copy/paste job here of things not needed for the test.

Reply:
done

Reply:
Is the variable `webdriver` ever assigned a value?